### DISCUSSION TITLE
Welcome to oauth2 Discussions!

### DISCUSSION THREAD

**Discussion by LyricTian** [go-oauth2/oauth2#256] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wesobi** [comment]
Hi @LyricTian 👋

I'm Wesley from Belgium and currently working on the IAM team at Showpad. Browsing in open source repositories here and there for new projects 👀.
